stellarphot/settings/models.py

```diff
@@ -815,18 +815,15 @@ class PassbandMap(BaseModelWithTableRep):
             examples=["Filter wheel 1"],
         ),
     ]
-    your_filter_names_to_aavso: list[PassbandMapEntry] | None
+    your_filter_names_to_aavso: list[PassbandMapEntry]
 
     def model_post_init(self, __context: Any) -> None:
         # Create a dictionary from the list of entries so that the object
         # can behave like a dictionary.
-        if self.your_filter_names_to_aavso is None:
-            self._dict = {}
-        else:
-            self._dict = {
-                entry.your_filter_name: entry.aavso_filter_name.value
-                for entry in self.your_filter_names_to_aavso
-            }
+        self._dict = {
```

Comment:
While reviewing code, found errors in the comments: 
line 774: "intialize" --> "initialize"
line 780: "interms" --> "in terms"

Otherwise, the big change here seems to be you don't allow initializing a `PassbandMap` with no `your_filter_names_to_aavso`.  OK.  I guess I am wondering if we should mention this in the documentation, essentially saying it behaves as an immutable dictionary once created. 

 I'll admit I also don't like the description of `your_filter_names_to_aavso` as `list[stellarphot.settings.PassbandMapEntry]` when in reality could be a list of `PassbandMapEntry` or a `dict`.   The documentation shows you can pass a `dict` in but not a list of `PassbandMapEntry` objects.  I sort of get why it has to be this way, but I had a hard time with the documentation.

Reply:
> I'll admit I also don't like the description of your_filter_names_to_aavso as list[stellarphot.settings.PassbandMapEntry] when in reality could be a list of PassbandMapEntry or a dict.

The kind of weird thing here is that for pydantic purposes (or really ipyautoui purposes) we need the type to be more specific than dict. ipyautoui doesn't know how to represent a dict (or at least it couldn't when I trtied a few months ago) in part because there are lots of ways you could in principle represent a `dict` in a ui. 

On the other hand, having to construct a `list[stellarphot.settings.PassbandMapEntry]` is a huge PITA....so allowing it to be set by a `dict` is an attempt to ease that pain for anyone providing the map programmatically.

I don't see a great fix...

Reply:
> While reviewing code, found errors in the comments:
> line 774: "intialize" --> "initialize"
> line 780: "interms" --> "in terms"

These are fixed now

Reply:
> Otherwise, the big change here seems to be you don't allow initializing a PassbandMap with no your_filter_names_to_aavso.

The change that goes with this is that `PassbandMap` is now optional (i.e. can be `None`) in the `PhotometrySettings`. That makes more sense to me than the original setup, where a `PassbandMap` was required but its value could essentially be `None`.